static/app/components/replays/replayView.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Panel, PanelBody, PanelHeader as _PanelHeader} from 'sentry/components/panels';
+import HorizontalMouseTracking from 'sentry/components/replays/player/horizontalMouseTracking';
+import {PlayerScrubber} from 'sentry/components/replays/player/scrubber';
+import ReplayController from 'sentry/components/replays/replayController';
+import ReplayCurrentUrl from 'sentry/components/replays/replayCurrentUrl';
+import ReplayPlayer from 'sentry/components/replays/replayPlayer';
+
+type Props = {
+  isFullscreen: boolean;
+  toggleFullscreen: () => void;
+};
+
+function ReplayView({isFullscreen, toggleFullscreen}: Props) {
+  return (
+    <PanelNoMargin isFullscreen={isFullscreen}>
+      <PanelHeader>
+        <ReplayCurrentUrl />
+      </PanelHeader>
+      <PanelHeader disablePadding>
+        <ManualResize isFullscreen={isFullscreen}>
+          <ReplayPlayer />
+        </ManualResize>
+      </PanelHeader>
+      <HorizontalMouseTracking>
+        <PlayerScrubber />
+      </HorizontalMouseTracking>
+      <PanelBody withPadding>
+        <ReplayController toggleFullscreen={toggleFullscreen} />
+      </PanelBody>
+    </PanelNoMargin>
+  );
+}
+
+const PanelNoMargin = styled(Panel)<{isFullscreen: boolean}>`
+  margin-bottom: 0;
+
+  ${p =>
+    p.isFullscreen
+      ? `height: 100%;
+      display: grid;
+      grid-template-rows: auto 1fr auto;
```

Comment:
This is a slight tweak that replaces the `<ReplayLayout>` styled from before.

basically in fullscreen the `<Url>` and `<Controller>` at top and bottom get their natural heigth, then the captured page in the middle takes up the rest.